src/main/java/de/rabea/game/Board.java

```diff
@@ -142,4 +153,19 @@ private Line getColumn(int index) {
         Line secondDiagonal = new Line(cells[2], cells[4], cells[6]);
         return Arrays.asList(firstDiagonal, secondDiagonal);
     }
+
+    public Mark winningPlayerMark() {
+        Mark winnerMark = null;
+        for (Line line : getAllLines()) {
+            if (line.hasWinner()) {
+                winnerMark = line.firstMarkInLine();
+            }
+        }
+        return winnerMark;
+    }
+
```

Comment:
+1 to Sarahs comment.
Something like
`getAllLines().stream().filter(line -> line.hasWinner).findFirst();`
Should return an `Optional<Mark>`.